Lab 1 Programs; CRC16, Bellman-Ford, LeakyBucket

anuragvaidya opened the pull request:
CRC16, Bellman-Ford and LeakyBucket algorithms in Java for the first networks lab.

anuragvaidya committed:
Lab 1 Programs; CRC16, Bellman-Ford, LeakyBucket

aksharsramesh commented on CRC16CCITT.java:
```diff
@@ -0,0 +1,36 @@
+        String s = sc.next();
+
+        sc.close();
+
```
Let us wait and check if she wants menu-driven or not
We can merge later tomorrow

ghpranav replied:
I agree with @aksharsramesh . I'll be pushing CPP programs as well.

ghpranav committed:
Lab 1: Add CPP programs
- Organise files into respective directories
- Add algorithms in README.md
- Add .gitignore

Signed-off-by: Pranav Bedre <bedrepranav@gmail.com>
